### DISCUSSION TITLE
editors#custom-editor-with-react-custom-element

### DISCUSSION THREAD

**Discussion by eswari379** [ghiscoding/slickgrid-react#277] [Q&A]
https://ghiscoding.gitbook.io/slickgrid-react/column-functionalities/editors#custom-editor-with-react-custom-element 

This document states that example 26 has an editor that includes a react element. However, I couldn't find Example 26 on the slickgrid-react demo page!
(https://github.com/ghiscoding/slickgrid-react/assets/37473247/04923a67-ca32-4b2a-a7ca-60ce079ec5c7)

Would you kindly provide the link to the SlickGrid Editor with React element?

**Comment by ghiscoding** [comment]
Ah right, these docs were copied from Aurelia-Slickgrid which does have an Example 26 but Slickgrid-React doesn't. The fact is that I'm not a React developer, someone else created this project and transferred it to me because I support all other SlickGrid repos including this one but the features related to creating React Component dynamically is not currently supported (since I don't know enough about React). There is a [reactUtil.ts](https://github.com/ghiscoding/slickgrid-react/blob/master/src/slickgrid-react/services/reactUtil.service.ts) that was created but is untested, the Aurelia-Slickgrid does have similar service used by Aurelia-Slickgrid [Example 26](https://ghiscoding.github.io/aurelia-slickgrid/#/example26). You could maybe take a look at how it's done in Aurelia-Slickgrid and see if it's working in Slickgrid-React (with the `reactUtil.ts` mentioned earlier)... for now I will simply remove the Example 26 from the docs since it's not currently supported. If you do get it working, please contribute back to the project.

**Comment by eswari379** [comment]
Thank you; I'll let you know when I'm through with it.
